Source/WebCore/testing/Internals.idl

```diff
@@ -1001,6 +1001,8 @@ typedef (FetchRequest or FetchResponse) FetchObject;
     DOMString resourceLoadStatisticsForURL(DOMURL url);
     undefined setTrackingPreventionEnabled(boolean enable);
 
+    undefined setSendPrivateTokenIPCForTesting(boolean enabled);
```

Comment:
How about using a UnifedWebPreference/Settings.
This way, InternalsSettings can be automatically generated and used.